packages/msupply/test/Adaptor.test.js

```diff
@@ -0,0 +1,132 @@
+import { expect } from 'chai';
+import { enableMockClient } from '@openfn/language-common/util';
+import { getItemsWithStats, insertOutboundShipment, request, upsertOutboundShipment } from '../src/Adaptor.js';
+import testData from './fixtures.json' assert {type: 'json'}
+
+
+
+const testServer = enableMockClient('https://fake.server.com');
+
+let state = {
+  configuration: {
+    "username": "username",
+    "password": "password",
+    "baseUrl": "https://fake.server.com"
+  }
+}
+
+describe('Get items with stats', () => {
```

Comment:
hmm. No action needed, but just a reflection that the tests on this adaptor should be focused on ensuring that the queries are built and assembled properly, not on the actual rest requests.

Because it's not interesting or risky that we can post to an endpoint. That stuff is well tested.

What's risky is that the query is invalid, or the wrong variables are passed, or something

It's just about thinking about what we're testing and why, and ensuring that the tests we add are valuable